src/core/evaluator.js

```diff
@@ -2162,13 +2162,16 @@ class PartialEvaluator {
     includeMarkedContent = false,
     sink,
     seenStyles = new Set(),
+    rotate = 0,
```

Comment:
Don't you want to include this parameter in the `getTextContent` invocation below as well?

https://github.com/mozilla/pdf.js/blob/f5b79be0b7fc2e56bdc9829b2c98369afe2e4e8e/src/core/evaluator.js#L2969-L2980